public/scripts/openai.js

```diff
@@ -1913,7 +1913,7 @@ async function sendOpenAIRequest(type, messages, signal) {
         'user_name': name1,
         'char_name': name2,
         'group_names': getGroupNames(),
-        'show_thoughts': Boolean(oai_settings.show_thoughts),
+        'include_reasoning': Boolean(oai_settings.show_thoughts),
```

Comment:
Any reason to change the name? Remember a lot of people might already have existing data with the original name chosen, and that leads to conflicts.
I don't see a big issue having internal name differences, and while `show_thoughts` give a *bit* of a wrong impression, it is now what it is.